courses/management/commands/gen_fake_data.py

```diff
@@ -30,8 +32,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         record_count = 0
+        try:
+            bi = BackingInstance.objects.get(instance_url='https://edx.org')
```

Comment:
`get_or_create` may be cleaner in this case since we don't do anything special in the `DoesNotExist` branch